packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -7702,6 +7702,18 @@ export default class Meeting extends StatelessWebexPlugin {
         ) {
           layoutInfo.content = {width: contentWidth, height: contentHeight};
         }
+
+        // @ts-ignore
+        this.webex.internal.newMetrics.submitClientEvent({
```

Comment:
This place here is only applicable to transcoded meetings. For multistream meetings, the layout is changed via RemoteMediaManager.setLayout() method. Also it doesn't feel right to be sending "client.media.render.start" at the point when we switch layout. Probably at this point we should be sending "client.share.layout.displayed" and "client.media.render.start" should I guess be sent when first frame is decoded and displayed. This will be tricky for SDK to know, because it doesn't have a reference to the video element that renders the stream and it only polls getStats() API every 5s so it can't immediately detect when frames are decoded - it may detect it up to 5s later. So maybe as a compromise we could send it when the share ReceiveSlot's sourceState changes to "live", but it still poses some challenges: 
- in theory there can be many share receive slots, which one should send it? the first one or all of them?
- ReceiveSlots are not used for transcoded meetings/calls